@here/harp-mapview/lib/FrustumIntersection.ts

```diff
@@ -142,19 +170,24 @@ export class FrustumIntersection {
 
                 const geoBox = getGeoBox(tilingScheme, childTileKey, offset);
 
+                // For tiles without elevation range source, default 0 will be propagated
```

Comment:
So if there is no elevationRangeSource you might want to check if the tile is already is in the cache and get the elevation from there.

Reply:
I have corrected comment here, it means that getGeoBox always returns box with altitude min/max equal to zero.